nexus-sdk/src/main/java/io/nexusrpc/handler/ServiceHandler.java

```diff
@@ -34,6 +39,16 @@ public Serializer getSerializer() {
     return serializer;
   }
 
+  private OperationHandler<Object, Object> getOperationHandler(
```

Comment:
```suggestion
  private OperationHandler<Object, Object> interceptOperationHandler(
```

Or "load" or "apply" or something, but this is pedantic/unimportant